apiary.apib

```diff
@@ -81,7 +81,73 @@ Any response that does not provide a ```200 OK``` response code should be treate
             
     + Attributes (Heartbeat)
 
+# Group Notifications
+
+## Request Collection [/notifications]
+
+### POST
+
++ Request (application/json)
+    + Attributes (Request Create)
+
++ Response 201 (application/json)
+    + Attributes (Request)
+
+## Request Detail [/notifications/{id}]
+
+### GET
+
++ Response 200 (application/json)
+    + Attributes (Request)
+
+## Message Collection [/notifications/{id}/messages]
```

Comment:
Missing all of the default paging & filtering parameters that are available.  You might also want to separate out the POST so that in the SDK you don't have to provide the query params